src/projectsGroups/index.ts

```diff
@@ -272,6 +272,70 @@ export class ProjectsGroups extends CrowdinApi {
         const url = `${this.url}/projects/${projectId}/file-format-settings/${fileFormatSettingsId}`;
         return this.patch(url, request, this.defaultConfig());
     }
+
+    /**
+     * @param projectId project identifier
+     * @param options optional parameters for the request
+     * @see https://developer.crowdin.com/api/v2/#operation/api.projects.strings-exporter-settings.getMany
+     */
+    listProjectStringExporterSettings(
```

Comment:
```suggestion
    listProjectStringsExporterSettings(
```


make it plural here and below (ProjectString**s**ExporterSettings)

Reply:
@kisharnath including the other methods, types and interfaces